lib/eu_central_bank.rb

```diff
@@ -209,13 +209,19 @@ def update_parsed_historical_rates(doc)
     @historical_last_updated = Time.now
   end
 
+
+  #new method for round
+  def exchange_with_round(from, to_currency, round = 2, date = nil)
```

Comment:
This will need to have a spec added, but let's figure out whether it's needed or not first